locale/de/LC_MESSAGES/django.po

```diff
@@ -245,43 +254,62 @@ msgstr "Du wirst zur Homepage weitergeleitet in"
 msgid "seconds..."
 msgstr "Sekunden..."
 
-#: templates/registration/password_reset_complete.html:5
+#: templates/registration/login.html:27
+#: templates/registration/password_reset_form.html:7
+msgid "Forgot password"
+msgstr "Passwort vergessen"
+
+#: templates/registration/password_change_done.html:5
+msgid "Your password has been changed"
+msgstr "Das Passwort wurde gesetzt"
+
+#: templates/registration/password_change_done.html:6
+#, fuzzy
```

Comment:
There should be no `fuzzy` message ids. This happens if the original message string is changed. Please revise the translation and if it is fine, remove this line.